com.interversehq.qview.yml

```diff
@@ -0,0 +1,21 @@
+app-id: com.interversehq.qview
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: qview
+rename-desktop-file: qView.desktop
```

Comment:
Ideally these renames should happen in your repository. As per freedesktop specs, icons, desktop launchers and appdata files should follow reverse DNS notation, just nothing enforces it.

Reply:
This should now be resolved, can you please attempt a new build, thanks. :)